pkg/logging/logging.go

```diff
@@ -202,6 +210,10 @@ func GetFormatter(format LogFormat) logrus.Formatter {
 			DisableTimestamp: true,
 			DisableColors:    true,
 		}
+	case LogFormatTextTimestamp:
+		return &logrus.TextFormatter{
+			DisableColors: true,
```

Comment:
This could have `DisableTimestamp: false`, similar to `LogFormatJSONTimestamp` below, for the sake of readability.

https://github.com/sirupsen/logrus/blob/dd1b4c2e81afc5c255f216a722b012ed26be57df/text_formatter.go#L50